x-pack/plugins/enterprise_search/public/applications/app_search/components/analytics/components/analytics_chart.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { useValues } from 'kea';
+
+import moment from 'moment';
+import { Chart, Settings, LineSeries, CurveType, Axis } from '@elastic/charts';
+
+import { KibanaLogic } from '../../../../shared/kibana';
+
+import { X_AXIS_DATE_FORMAT, TOOLTIP_DATE_FORMAT } from '../constants';
+
+interface Props {
```

Comment:
Non-blocking feedback:

Could data be a type?

```ts
interface Point {
  x: string;
  y: number;
}

interface Props {
  height?: number;
  lines: Array<{
    id: string;
    data: Array<Point>;
  }>;
```

`convertToChartsData` could use that for it's return type as well.

Reply:
Sure, no objections! I'll probably export `interface Point` from the chart component file - I'm inclined to keep it there because that's a format very specific to Elastic Charts

Reply:
eb510b7